frontend/src/components/ai-elements/task.tsx

```diff
@@ -77,9 +77,11 @@ export const TaskContent = ({
 }: TaskContentProps) => (
   <CollapsibleContent
     className={cn(
-      "data-[state=closed]:fade-out-0 data-[state=closed]:slide-out-to-top-2 data-[state=open]:slide-in-from-top-2 text-popover-foreground outline-none data-[state=closed]:animate-out data-[state=open]:animate-in",
+      "text-popover-foreground outline-none",
       className
     )}
+    transition={{ duration: 0 }}
+    style={{ layout: 'position' } as any}
```

Comment:
Can we refactor the `style` prop to use tailwind classnames and remove the `as any` directive?